docs/modules/operation/pages/events/event-configuration.adoc

```diff
@@ -203,39 +93,12 @@ When slotting your own event definitions, take care not to place them below thes
 * To save memory and shorten startup times, you may wish to remove event definition files that you know you do not need.
 * If you need to customize some events in one of the default tributary files, you may wish to make a copy of the file containing only the customized events, and slot the copy above the original; this practice will make it easier to maintain your customizations in case the default file changes in a future release of {page-component-title}.
```

Comment:
```suggestion

* To save memory and shorten startup times, you may want to remove event definition files that you know you do not need.
* If you need to customize some events in one of the default tributary files, you may want to make a copy of the file containing only the customized events and load the copy above the original in `eventconf.xml`.
This practice will make it easier to maintain your customizations in case the default file changes in a future release of {page-component-title}.
```

Reply:
@emarsh1995 What do you think about this tweak of the text?

Reply:
I like it! Breaking the second bullet point into two sentences is a good idea. :)